src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -331,50 +323,22 @@ void Profile::LayerJson(const Json::Value& json)
     JsonUtils::GetValueForKey(json, StartingDirectoryKey, _StartingDirectory);
 
     JsonUtils::GetValueForKey(json, IconKey, _Icon);
-    JsonUtils::GetValueForKey(json, BackgroundImageKey, _BackgroundImagePath);
-    JsonUtils::GetValueForKey(json, BackgroundImageOpacityKey, _BackgroundImageOpacity);
-    JsonUtils::GetValueForKey(json, BackgroundImageStretchModeKey, _BackgroundImageStretchMode);
-    JsonUtils::GetValueForKey(json, BackgroundImageAlignmentKey, _BackgroundImageAlignment);
-    JsonUtils::GetValueForKey(json, RetroTerminalEffectKey, _RetroTerminalEffect);
     JsonUtils::GetValueForKey(json, AntialiasingModeKey, _AntialiasingMode);
     JsonUtils::GetValueForKey(json, TabColorKey, _TabColor);
     JsonUtils::GetValueForKey(json, BellStyleKey, _BellStyle);
-    JsonUtils::GetValueForKey(json, PixelShaderPathKey, _PixelShaderPath);
-}
 
-// Method Description:
-// - Either Returns this profile's background image path, if one is set, expanding
-// - Returns this profile's background image path, if one is set, expanding
-//   any environment variables in the path, if there are any.
-// - Or if "DesktopWallpaper" is set, then gets the path to the desktops wallpaper.
-// Return Value:
-// - This profile's expanded background image path / desktops's wallpaper path /the empty string.
-winrt::hstring Profile::ExpandedBackgroundImagePath() const
-{
-    const auto path{ BackgroundImagePath() };
-    if (path.empty())
-    {
-        return path;
-    }
-    // checks if the user would like to copy their desktop wallpaper
-    // if so, replaces the path with the desktop wallpaper's path
-    else if (path == DesktopWallpaperEnum)
+    if (json.isMember(JsonKey(UnfocusedAppearanceKey)))
     {
-        WCHAR desktopWallpaper[MAX_PATH];
+        auto unfocusedAppearance{ winrt::make_self<implementation::AppearanceConfig>() };
 
-        // "The returned string will not exceed MAX_PATH characters" as of 2020
-        if (SystemParametersInfo(SPI_GETDESKWALLPAPER, MAX_PATH, desktopWallpaper, SPIF_UPDATEINIFILE))
-        {
-            return winrt::hstring{ (desktopWallpaper) };
-        }
-        else
-        {
-            return winrt::hstring{ L"" };
-        }
-    }
-    else
-    {
-        return winrt::hstring{ wil::ExpandEnvironmentStringsW<std::wstring>(path.c_str()) };
+        // If an unfocused appearance is defined in this profile, any undefined parameters are
+        // taken from this profile's default appearance, so add it as a parent
+        com_ptr<AppearanceConfig> parentCom;
+        parentCom.copy_from(defaultAppearanceImpl);
+        unfocusedAppearance->InsertParent(parentCom);
+
+        unfocusedAppearance->LayerJson(json[JsonKey(UnfocusedAppearanceKey)]);
+        _UnfocusedAppearance = *unfocusedAppearance;

```

Comment:
I feel like we need a big ASCII block diagram in the comments that explains this. I keep reading it and re-reading it and i *THINK* i understand it?

Reply:
But the unfocused appearance itself _is_ inheritable? So Profile A can inherit Defaults.Unfocused, and Defaults.Unfocused inherits Defaults

Reply:
Yes! The way I think about it is if I ask profile X for its ``unfocusedAppearance`, there are 3 possible scenarios:

- profile X has one defined and returns it (this one inherits from profile X)
- profile X does not have one defined, but defaults does, so we get the one that defaults defined (this one inherits from defaults)
- neither X nor defaults define one, so we get nothing

So if someone has `unfocusedAppearance` defined in `defaults`, but wants profile Y to look the same when focused and when unfocused, they will need to define an empty `unfocusedAppearance` in profile Y (similar to if they have a `backgroundImage` defined in defaults but want one of their profiles to have no `backgroundImage`)